index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">The Rock</h1>
+         
+         <div id="img-div">
+
+            <img src="https://www.biography.com/.image/ar_1:1%2Cc_fill%2Ccs_srgb%2Cfl_progressive%2Cq_auto:good%2Cw_1200/MTc5NjIyODM0ODM2ODc0Mzc3/dwayne-the-rock-johnson-gettyimages-1061959920.jpg" alt="" id="image">
+            <p id="img-caption">The Rock posing intimidately.</p>
+         </div>
+
+         <p id="tribute-info">
+             The Rock is a former WWE fighter that really wanted to make it into
+             Hollywood (and made it actually). He has like a million movies and he is
+             the most followed actor in Instagram.
+        </p>
+       <p>
+           If you want to know more about The Rock just go <a href="https://fr.wikipedia.org/wiki/Dwayne_Johnson"  target="_blank" id="tribute-link">here</a>
```

Comment:
please fix the linter error relate to the link with attribute target="__blank" by adding rel="noreferrer" rel="noopener"
attributes without those attribute your site may encounter security issues and bad performance.